deploy/Makefile

```diff
@@ -0,0 +1,22 @@
+LIKED_VERSION ?= $(shell git describe --tags --abbrev=0 | sed 's|v\(.*\)|\1|g')
+COSMOVISOR_VERSION ?= "1.1.0"
+MONIKER ?= oursky
+
+LIKED_WORKDIR ?= $(HOME)
+LIKED_HOME ?= "$(LIKED_WORKDIR)/.liked"
+LIKED_USER ?= "$(USER)"
+
+.PHONY: setup-node
+setup-node:	
+	LIKED_VERSION=$(LIKED_VERSION) COSMOVISOR_VERSION=$(COSMOVISOR_VERSION) ./scripts/node-setup.sh $(MONIKER) $(GENESIS_URL) $(LIKED_WORKDIR) $(LIKED_HOME) $(LIKED_USER)
+
+.PHONY: initialize-systemctl
+initialize-systemctl: 
+	sudo cp ./liked.service /etc/systemd/system/liked.service
+	sudo systemctl daemon-reload
+	sudo systemctl start liked
```

Comment:
this will start cosmovisor immediately? better make service start a user initiated action to avoid double sign?